destinationCreation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8"/>
+      <title>Destination Creation</title>
+      <link rel="stylesheet" href="destinationCreation.css"/>
+      <script src="destinationCreation.js"></script>
+    </head>
+    <header>
+    </header>
+    <body>
+      <div id="content">
+        <h1>Destination Creation Form</h1>
+        <div id="form-container">
+          <form action="/data" method="post">
```

Comment:
/data will be the path for the POST request, so I think giving it a more descriptive name like /create-destination might make more sense